keanu-project/src/test/java/io/improbable/keanu/e2e/eight/EightSchools.java

```diff
@@ -0,0 +1,67 @@
+package io.improbable.keanu.e2e.eight;
+
+import io.improbable.keanu.algorithms.NetworkSamples;
+import io.improbable.keanu.algorithms.mcmc.NUTS;
+import io.improbable.keanu.network.BayesianNetwork;
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.KeanuRandom;
+import io.improbable.keanu.vertices.dbl.probabilistic.GaussianVertex;
+import io.improbable.keanu.vertices.dbl.probabilistic.HalfCauchyVertex;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+
+@Slf4j
+public class EightSchools {
+
+    public static void main(String[] args) {
+        EightSchools model = new EightSchools();
+        model.run();
+    }
+
+    public void run() {
+        KeanuRandom random = new KeanuRandom();
+
+        int J = 8;
+        double[] yObs = new double[]{28., 8., -3., 7., -1., 1., 18., 12.};
+        double[] sigma = new double[]{15., 10., 16., 11., 9., 11., 10., 18.};
+
+        DoubleVertex mu = new GaussianVertex(0, 5);
+        mu.setValue(0);
+        DoubleVertex tau = new HalfCauchyVertex(5);
+        tau.setValue(3);
+        DoubleVertex theta_tilde = new GaussianVertex(new long[]{1, J}, 0, 1);
```

Comment:
what's `theta_tilde`? Plus it should be camel case

Reply:
It's from the example but we should at least camelCase it.